src/button/template/componentTemplate.jsx

```diff
@@ -83,15 +83,18 @@ function renderFundingIcons({ cards, fundingicons } :
     return <div class={ `${ CLASS.FUNDINGICONS }` }>{ renderCards({ cards, button: true }) }</div>;
 }
 
-function renderContent(text : string, { locale, color, branding, logoColor, funding, env, cards } :
-    { locale : LocaleType, color : string, branding? : boolean, logoColor? : string, funding? : FundingSelection, env : string, cards : Array<string> }) : JsxHTMLNode {
+function renderContent(text : string, { label, locale, color, branding, logoColor, funding, env, cards, dynamicContent } :
+    { contentLabel?: string, locale : LocaleType, color : string, branding? : boolean, logoColor? : string, funding? : FundingSelection, env : string, cards : Array<string>, dynamicContent : Object }) : JsxHTMLNode {
 
     let content = getLocaleContent(locale);
 
     return jsxRender(text, {
 
         text(value : string) : JsxHTMLNode {
```

Comment:
Would it be possible to add a new key/function to `jsxRender`, so we don't overload `text`? Then we could have support for `foo {break} bar`, and the parser that's already there would take care of regexing the string and splitting it up. It would also mean we'd be able to use multiple breaks in a string if we ever need to.

Reply:
I would need to check the feasibility for that. jsxRender function is tightly coupled with the current pattern of content.

Reply:
Yes, added a separate break function.